.github/workflows/python-package.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       matrix:
         mongodb-version: [4.0.18]
         python-version: [3.8, 3.9]
-        java-version: [1.8]
+        java-version: [1.11]
```

Comment:
Any particular reason for this? Currently, our Java code is not certified to work with anything above 8.

Reply:
Yes, this is just for Nextflow, I think it must have something to do with the setup on Github Actions specifically as we're obviously able to run Nextflow with Java 8 usually...